### PR TITLE
TELCODOCS-781: Factory precaching cli tool ZTP integration

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.12, 4.13+

Issue: https://issues.redhat.com/browse/TELCODOCS-781

Link to docs preview: 
* https://54289--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-precaching-tool.html#ztp-pre-caching-config-con_pre-caching
* https://54289--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-precaching-tool.html#ztp-pre-staging-troubleshooting_pre-caching

QE review:
- [x] QE has approved this change.

Additional information: z-stream release

### CODE REVIEW

File: modules/ztp-precaching-troubleshooting.adoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+level=info msg=trying next host error=failed to do request: Head "https://eko4.cloud.lab.eng.bos.redhat.com:8443/v2/redhat/redhat-operator-index/manifests/v4.11": x509: certificate signed by unknown authority host=eko4.cloud.lab.eng.bos.redhat.com:8443
+
+The rendered catalog is invalid.
+
+Run "oc-mirror list operators --catalog CATALOG-NAME --package PACKAGE-NAME" for more information.
+
+error: error rendering new refs: render reference "eko4.cloud.lab.eng.bos.redhat.com:8443/redhat/redhat-operator-index:v4.11": error resolving name : failed to do request: Head "https://eko4.cloud.lab.eng.bos.redhat.com:8443/v2/redhat/redhat-operator-index/manifests/v4.11": x509: certificate signed by unknown authority
+----
+
+. To solve this issue, first, copy the registry certificate into your server:
```
Comment (sheriff-rh) [inline modules/ztp-precaching-troubleshooting.adoc:38]: Additionally, start your procedural steps with the verb: 

"Copy the registry cert..."

**Review by sheriff-rh — COMMENTED** [review]
Looks great overall - some comments and suggestions for improvement :)

**Comment by obochan-rh** [comment]
lgtm

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**